rules/aip0191/java_outer_classname.go

```diff
@@ -29,7 +29,11 @@ var javaOuterClassname = &lint.FileRule{
 	OnlyIf: hasPackage,
 	LintFile: func(f *desc.FileDescriptor) []lint.Problem {
 		if f.GetFileOptions().GetJavaOuterClassname() == "" {
-			want := strcase.UpperCamelCase(strings.ReplaceAll(f.GetName(), ".", "_"))
+			filename := f.GetName()
+			if ix := strings.LastIndex(filename, "/"); ix != -1 {
```

Comment:
Should we use `filepath.Base` instead?

Reply:
I did not know that was a thing. Thanks!